### ISSUE TITLE
Edit resources

### ISSUE THREAD

**Issue by jacksgt** [jinnovation/kele.el#157]
Hello!

I came here after trying `kubernetes.el` which unfortunately support only a very limited number of Kubernetes resources - so thanks for this cool project!

As far as I can tell editing resources (similar to what can be done in `kubernetes.el`) is not supported right?

I believe this is a pretty important feature and would drastically improve the usability of `kele`.

**Comment by jinnovation** [comment]
Agree, I think this would be pretty fantastic. Thanks for opening this, let's drop notes on this issue re: implementation details in case anybody has bandwidth to pick this up.

**Comment by jinnovation** [comment]
Started a reference implementation over on branch `157/edit-resource`. `kele-kubectl-do` doesn't quite seem to work for this for reasons I have yet to figure out.

**Comment by jacksgt** [comment]
Awesome! I think the user workflow shouldn't be more complicated than navigating the cursor onto the line of a particular resource (within `kele`), hitting Enter and be dropped into a YAML buffer. From there the user can `C-c C-c` to apply the changes or `C-c C-k` to discard (this would match the `magit` commit buffer behavior).

**Comment by jinnovation** [comment]
> Awesome! I think the user workflow shouldn't be more complicated than navigating the cursor onto the line of a particular resource (within `kele`), hitting Enter and be dropped into a YAML buffer. From there the user can `C-c C-c` to apply the changes or `C-c C-k` to discard (this would match the `magit` commit buffer behavior).

I like the Magit-style user flow here. This feels a lot better than my initial thinking here, which was to simply call `kubectl edit` in a sub-process -- both in terms of UX as well as ease of implementing. This would also fold in nicely w/ existing flows, allowing user to take the resource buffer retrieved via `kele-get`, edit it, then `C-c C-c` to apply the new manifest.
